src/DivanteClipboardBundle.php

```diff
@@ -6,6 +6,11 @@
 
 class DivanteClipboardBundle extends AbstractPimcoreBundle
 {
+    public function getInstaller()
+    {
+        return new Installer();
```

Comment:
https://pimcore.com/docs/5.0.x/Extending_Pimcore/Bundle_Developers_Guide/Pimcore_Bundles/Installers.html

> It's recommended to define the installer as service and to fetch it from the container from your bundle class on demand.

Moreover, I think that it will be good to have here phpdoc like:
```
/**
 * @return Installer
 */
```

Reply:
Done